App/HejDTO/Sources/HejDTO/Contests/GET/ContestDetail/Contests.Details.swift

```diff
@@ -0,0 +1,26 @@
+
+import Foundation
+
+extension Contests {
+    
+    public struct ContestDetail: Codable, Equatable {
```

Comment:
pleas add url to the documentation :)

Reply:
Added :)

Reply:
Also just `Detail` for the name should be ok :) Please check other types for the same issue (repeated "upper" type name) :)

Reply:
I hope it should be fine now :)